src/arch/ArchHooks/ArchHooks_Win32Static.cpp

```diff
@@ -9,38 +9,48 @@
 #include <cstdint>
 #include <vector>
 
-// for timeGetTime
+// for QueryPerformanceCounter
 #include <windows.h>
 #include <mmsystem.h>
 #if defined(_MSC_VER)
 #pragma comment(lib, "winmm.lib")
 #endif
 
-
 static bool g_bTimerInitialized;
 
+/* QueryPerformanceCounter variables below.
+ * QueryPerformanceCounter and QueryPerformanceFrequency expect
+ * a LARGE_INTEGER, which is a union. These functions store data
+ * in the QuadPart of the LARGE_INTEGER, which is type longlong. */
+LARGE_INTEGER g_bFrequency, g_bStartTime, g_bCurrentTime;
+double elapsedTime;
```

Comment:
I'm reading https://learn.microsoft.com/en-us/windows/win32/sysinfo/acquiring-high-resolution-time-stamps and I don't think we need the `elapsedTime` to be a double. Seems like we're taking a int64, converting it to a double, and then converting it back to an int64. I think we can delete the elapsedTime def altogether and then further changes below.